src/i18n/i18n.ts

```diff
@@ -20,7 +20,11 @@ export const availableLanguages = Object.keys(resources);
 i18n.use(LanguageDetector).use(initReactI18next).init({
   resources,
   fallbackLng: "en",
+
+  // allow keys to be phrases having `:`, `.`
+  nsSeparator: false,
   keySeparator: false,
+
```

Comment:
Not sure there is a reason you spaced things out a bit, but since they are an object being passed in to `init`, please keep things tight.
```suggestion
  // allow keys to be phrases having `:`, `.`
  nsSeparator: false,
  keySeparator: false,
```